### PR TITLE
filmic + colorbalance: update presets

### PR DESCRIPTION
**Pull request by aurelienpierre**
colorbalance: remove the presets that are a burden to maintain (assert size error on the mask & blendop)
filmic: update default parameters to more usual values

### CODE REVIEW

File: src/iop/colorbalance.c [hunk 1]
```diff
@@ -233,50 +233,6 @@ int legacy_params(dt_iop_module_t *self, const void *const old_params, const int
...
-    dt_gui_presets_add_with_blendop(name, self->op, self->version(),
-                                    p, len, bp, 1);
-  free(bp);
-  free(p);
-}
-
-void init_presets(dt_iop_module_so_t *self)
-{
-  // these blobs were exported as dtstyle and copied from there:
-  add_preset(self, _("split-toning teal-orange (2nd instance)"),
```
Comment (TurboGit) [inline src/iop/colorbalance.c]: Too bad :( I've not followed the discussion, but the blendop legacy support does not work?

Comment (TurboGit) [reply src/iop/colorbalance.c]: I mean we have the same circuitry in denoisprofile for example where it seems to work, no?

File: src/iop/colorbalance.c [hunk 2]
```diff
@@ -233,50 +233,6 @@ int legacy_params(dt_iop_module_t *self, const void *const old_params, const int
...
-    else
-    {
-      free(bp);
-      free(bp_new);
-      bp = NULL;
-    }
-  }
-
-  if(p && bp)
-    dt_gui_presets_add_with_blendop(name, self->op, self->version(),
```
Comment (TurboGit) [inline src/iop/colorbalance.c]: self->version() here seems strange, it means that the version of the param is always set to be the latest version of the module. It should be the version of the param at the time the preset param was created I think?

File: src/iop/colorbalance.c (hunk 1 above)
Comment (aurelienpierre) [reply src/iop/colorbalance.c]: not for hard-coded presets it seems. Everytime the blendop API is updated, the presets become invalid and raise a size error

File: src/iop/colorbalance.c (hunk 2 above)
Comment (aurelienpierre) [reply src/iop/colorbalance.c]: That might be the problem. Honestly, I just took that code from the denoiseprofile IOP, I didn't give it a lot of thought

Comment (TurboGit) [reply src/iop/colorbalance.c]: The code in denoiseprofile do use version which is passed to the add_preset().

File: src/iop/colorbalance.c (hunk 1 above)
Comment (aurelienpierre) [reply src/iop/colorbalance.c]: anyway, the concept of having static blobs exported from somewhere else in the code is not the cleanest design.

**Comment by aurelienpierre** [comment]
I will try to find a better option

**Comment by TurboGit** [comment]
@aurelienpierre, see my fix it was indeed the params itself the issue using wrong version and not the blend params which were properly handled.

**Pull request closed without merging**